### ISSUE TITLE
Workstream definition: Adaptive Profile specification 

### ISSUE THREAD

**Issue by bettinaheim** [qir-alliance/.github#32]
This issue is created to capture any discussions or feedback also from the broader community regarding the creation of a workstream to specify an Adaptive Profile. The workstream definition document added with PR #31 captures the goals, requirements, expected outcomes, as well as what is in and out of scope for the workstream. Please comment on this issue to share thoughts about the workstream itself (i.e. about the content of that document). A suitable issue on the spec repo to track the work for defining such an Adaptive Profile will be created once the workstream goals are clear and the workstream is approved.
